src/scenes/home/signup/signup.css

```diff
@@ -1,3 +1,13 @@
+
+.signup select {
+  height: 4.2rem;
+  width: 100%;
+  padding: 2rem;
```

Comment:
The vertical padding is preventing the selected option from being visible on Chrome for Windows. Recommend changing the padding to 0 16px (or the rem equivalent) to mirror the styling on the form inputs.